### PR TITLE
LPD-21979 DM Filter&Search put under Feature Flag 

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Missed FF for the epic https://liferay.atlassian.net/browse/LPD-11313. 
The only story affected (is the only only developed) is https://liferay.atlassian.net/browse/LPD-6878

### CODE REVIEW

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java [hunk 1]
```diff
@@ -1088,8 +1088,13 @@ protected int processStartTag() throws Exception {
 				jspWriter.write("\"");
 			}
 
+			String searchPlaceholderKey =
+				FeatureFlagManagerUtil.isEnabled("LPD-11313") ? "search" :
+					"search-for";
+
```
Comment (adolfopa) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java:1094]: This is only used once. Can you inline it?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java]: Thanks for reviewing this @adolfopa. I've inlined it in https://github.com/liferay-content-management/liferay-portal/pull/5297/commits/1841ff65bca023e27e6fd9d2341f31dfba987a5a

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:report:2559169061

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by ambrinchaudhary** [comment]
ci:report:2559224213

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by ambrinchaudhary** [comment]
Erros does not seem related. Not sure what should I do here.... 

I'm going to close this pull and open another one, and see if I'm lucky 🤞

**Pull request closed without merging**